doc/ug/dv_methodology.md

```diff
@@ -415,6 +415,28 @@ It is recommended to follow these guidelines when collecting coverage:
 *  Collect all coverage metrics (except toggle based on above bullet) on the DUT and all of its non-pre-verified sub-modules
 *  Treat pre-verified sub-modules as ['black-box'](#integration-testing) in terms of coverage collection
 
+### X-Propagation (Xprop)
+
+It's mandatory to enable Xprop when running regression for coverage closure.
+Xprop is enabled by default and we choose `tmerge` mode which is closer to actual hardware behavior and is the more commonly used mode.
+It's recommended to enable Xprop all the time since it only adds less than 10% simulation time.
+
+In order to test X propagation more effectively, we should implement driver to set input data/address/control signals to X when it's not in valid period.
```

Comment:
To test X-prop more effectively, the address / data / control signals are required to be driven to Xs when invalid (`valid` bit is not set).

Reply:
Done